catalog/core/catalog-core-api/pom.xml

```diff
@@ -90,7 +90,7 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>common-system</Embed-Dependency>
                         <Export-Package>
-                            ddf.catalog*;version=2.0.12
+                            ddf.catalog*
```

Comment:
Should we be setting version to ${project.version} here? Otherwise this is going to be exported as version 0.0.0. I'm not sure why it was hardcoded to 2.0.12 before, that also doesn't seem correct.